cmd/build/v1/build.go

```diff
@@ -59,7 +60,11 @@ func NewBuilder(builder OktetoBuilderInterface, registry oktetoRegistryInterface
 
 // NewBuilderFromScratch creates a new okteto builder
 func NewBuilderFromScratch(ioCtrl *io.IOController) *OktetoBuilder {
```

Comment:
This builder uses okteto context global var because is the builder constructor used by other commands which do not have the context injected. As each command is refactored to remove the global context, each command should use a constructor that injects the context. `okteto build` command does not uses this logic,